portafly/src/components/data-list/reducers/BulkActionsReducer.ts

```diff
@@ -8,15 +8,25 @@ import {
 export type BulkAction = 'sendEmail' | 'changeState' | undefined
 export type BulkActionsState = {
   modal: BulkAction,
-  isLoading: boolean
+  isLoading: boolean,
+  errorMsg?: string
+}
+
+const defaultBulkActions: BulkActionsState = {
+  modal: undefined,
+  isLoading: false,
+  errorMsg: undefined
 }
 
 // Action Handlers
-const BulkActionsActionHandlers: ActionHandlers<BulkActionsState, BulkAction> = {
+const BulkActionsActionHandlers: ActionHandlers<BulkActionsState, any> = {
```

Comment:
the `any` is because it has troubles when it's undefined?

Reply:
It's because it can be either BulkAction, string or undefined

Reply:
Hmm we will need to review all the types we are using in our reducers afterwards